src/main/java/com/sarahhand/fractals/model/colorscheme/ColorScheme.java

```diff
@@ -2,14 +2,25 @@
 
 import java.awt.Color;
 
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.sarahhand.fractals.model.FractalConfig;
 import com.sarahhand.fractals.model.PointData;
-
+import com.sarahhand.fractals.model.colorscheme.mandelbrotset.BlackColorScheme;
+import com.sarahhand.fractals.model.colorscheme.mandelbrotset.EscapeTimeColorScheme;
 
 /**
  * Interface that represents ways of coloring fractals.
  * @author J9465812
  */
+@JsonTypeInfo(
+        use = JsonTypeInfo.Id.NAME,
+        include = JsonTypeInfo.As.PROPERTY,
+        property = "type")
+@JsonSubTypes({
+    @JsonSubTypes.Type(value = EscapeTimeColorScheme.class, name = "EscapeTime"),
+    @JsonSubTypes.Type(value = BlackColorScheme.class, name = "Black") 
```

Comment:
Don't forget to add the subtype ExternalDistanceEstimateColorScheme once my pull request has been merged.